src/os/bluestore/AvlAllocator.h

```diff
@@ -50,6 +50,7 @@ class AvlAllocator final : public Allocator {
 public:
   AvlAllocator(CephContext* cct, int64_t device_size, int64_t block_size,
 	       const std::string& name);
+  virtual ~AvlAllocator();
```

Comment:
i'd suggest put 
```c++
~AvlAllocator() final;
```
instead.

Reply:
@tchaikov Note. I do not know why I added "virtual" before destructor. It would be ok for pascal, but not here. My mind just broke for a minute.